packages/camera/camera_avfoundation/example/ios/RunnerTests/CameraPluginCreateCameraTests.swift

```diff
@@ -0,0 +1,127 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import XCTest
+
+@testable import camera_avfoundation
+
+final class CameraPluginCreateCameraTests: XCTestCase {
+  private func createSutAndMocks() -> (
```

Comment:
what's Sut?

Reply:
I believe it stands for "system under test." I assumed it was an existing naming convention in this plugin, but it seems @FirentisTFW started it. I can rename it if it's unclear

Reply:
Can you rename to `createCameraPlugin`